helpers/helpers.go

```diff
@@ -93,3 +93,21 @@ func PersistExistingParams(values []string, key, ignoreValue string, q url.Value
 func ToBoolPtr(val bool) *bool {
 	return &val
 }
+
+// GetOfficialStatisticsLogo returns the official statistics logo based on the enableOfficialStatisticsLogo and language
+func GetOfficialStatisticsLogo(enableOfficialStatisticsLogo bool, language string, useSvg bool) string {
```

Comment:
This function could do with unit tests to confirm the given parameters return the expected results